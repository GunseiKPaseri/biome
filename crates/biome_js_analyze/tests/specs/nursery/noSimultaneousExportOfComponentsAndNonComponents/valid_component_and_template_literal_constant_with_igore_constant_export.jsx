const foo = 'world'
export const CONSTANT = `Hello ${foo}`
export const Foo = () => {}
