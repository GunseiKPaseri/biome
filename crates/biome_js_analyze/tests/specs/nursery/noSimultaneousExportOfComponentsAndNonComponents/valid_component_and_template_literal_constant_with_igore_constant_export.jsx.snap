---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: valid_component_and_template_literal_constant_with_igore_constant_export.jsx
---
# Input
```jsx
const foo = 'world'
export const CONSTANT = `Hello ${foo}`
export const Foo = () => {}

```
